src/setupTests.js

```diff
@@ -1,5 +0,0 @@
-// jest-dom adds custom jest matchers for asserting on DOM nodes.
-// allows you to do things like:
-// expect(element).toHaveTextContent(/react/i)
-// learn more: https://github.com/testing-library/jest-dom
```

Comment:
- [ ] As it is a requirement for this project under technical set up, and bests practice to ensure your app is not broken by new changes, please kindly add tests. See screenshots below and kindly make appropriate changes. 

![test](https://user-images.githubusercontent.com/65852621/142737065-9bfccdd1-11af-4e28-b034-9d8d7be7ff6f.png)

![test2](https://user-images.githubusercontent.com/65852621/142737102-a51cb9c0-14e5-4b49-8c93-ade42a312ad3.png)